glue-exp/importers/vizier/vizier.py

```diff
@@ -0,0 +1,77 @@
+# This plugin requires OpenCV to be installed, and returns a PIL Image object
+
+import os
+import sys
+from io import BytesIO
+
+import requests
+from astropy.io.votable import parse, parse_single_table
+
+from PyQt4.QtGui import QDialog, QApplication, QTreeWidgetItem, QHeaderView
+from PyQt4.uic import loadUi
+from PyQt4.QtCore import Qt
+
+UI_MAIN = os.path.join(os.path.dirname(__file__), 'vizier.ui')
+
+
+class VizierImporter(QDialog):
+
+    def __init__(self):
+
+        super(VizierImporter, self).__init__()
+
+        self.ui = loadUi(UI_MAIN, self)
+
+        self.cancel.clicked.connect(self.reject)
```

Comment:
Minor style nit. This isn't super relevant for this simple class.

There are a collection of classes in `glue.qt.widget_properties` that allow you to wrap common buttons in properties, and also to attach state callback functions to widget state changes. I find that it makes the code cleaner in a few important ways:

1) It hides a lot of the QT-specific API behind the more generic abstraction of python properties
2) It cuts down on boilerplate code for setting widget state and connecting signals
3) It makes the state of the widget easier to understand (and save/restore). Usually widget state can be fully described by the settings of all the widget properties.

I also try to make the GUI-specific parts of code as "dumb" as possible, since they are harder to test. Ideally GUI code lives in a class that solely exists to define widgets, tweak appearance, and connect signal handlers. So for example I might extract the VO parsing step into it's own function outside of the class, and have the GUI only know how to populate a table when given a list of (description, nrows, name) tuples.